src/view/components/rule-engine/rule-engine.ts

```diff
@@ -17,8 +17,8 @@ const I8N_BASE:string = 'api.sites.ruleengine'
   selector: 'cw-rule-engine'
 })
 @View({
-  template: `<div flex layout="column" class="cw-rule-engine">
-  <div flex layout="column" class="cw-header">
+  template: `<div class="cw-rule-engine">
+  <div class="cw-header">
     <div flex layout="row" layout-align="space-between center">
```

Comment:
For this case I can definitely see your argument - block parent, row-layout child, flex grandchildren.